src/overlays/actors/ovl_En_Crow/z_en_crow.c

```diff
@@ -21,41 +30,446 @@ const ActorInit En_Crow_InitVars = {
+        }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E06E8.s")
+        func_8002E4B4(globalCtx, &this->actor, 12.0f * scale, 25.0f * scale, 50.0f * scale, 7);
+    } else {
+        height = 0.0f;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0770.s")
+    this->collider.list[0].dim.worldSphere.center.x = this->actor.posRot.pos.x;
+    this->collider.list[0].dim.worldSphere.center.y = this->actor.posRot.pos.y + height;
+    this->collider.list[0].dim.worldSphere.center.z = this->actor.posRot.pos.z;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0864.s")
+    if (this->actionFunc == func_809E0C8C) {
+        CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0C8C.s")
+    if (this->collider.base.acFlags & 1) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0E2C.s")
+    if (this->actionFunc != func_809E10A8) {
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E0F54.s")
+    Actor_SetHeight(&this->actor, height);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E1004.s")
+    if (this->actor.colChkInfo.health != 0 && func_800A56C8(&this->skelAnime, 3.0f)) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_KAICHO_FLUTTER);
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E10A8.s")
+s32 EnCrow_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                            Actor* thisx) {
+    EnCrow* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E1174.s")
+    if (this->actor.colChkInfo.health != 0) {
+        if (limbIndex == 7) {
+            rot->y += 0xC00 * sinf(this->skelAnime.animCurrentFrame * (M_PI / 4));
+        } else if (limbIndex == 8) {
+            rot->y += 0x1400 * sinf((this->skelAnime.animCurrentFrame + 2.5f) * (M_PI / 4));
+        }
+    }
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/EnCrow_Update.s")
+void EnCrow_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnCrow* this = THIS;
+    Vec3f* vec;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Crow/func_809E1434.s")
+    if (limbIndex == 2) {
+        Matrix_MultVec3f(&sHeadVec, &this->firePos[0]);
+        this->firePos[0].y -= 20.0f;
+    } else if (limbIndex == 4 || limbIndex == 6 || limbIndex == 8) {
+        vec = &this->firePos[(limbIndex >> 1) - 1];
+        Matrix_MultVec3f(&sZeroVecAccel, vec);
+        vec->y -= 20.0f;
+    }
```

Comment:
Considering this usage, the array you called `firePos` should probably be a generic `bodyPartsPos` array, used as the positions for specific body parts. This is probably true for any actor that gets burned with `EffectSsEnFire`.